Tiltfile

```diff
@@ -1,363 +1,363 @@
-        print("About to start {} with command {}".format(name, cmd))
-
-    local_resource(name, "", auto_init=auto_init, serve_cmd=cmd, serve_env=env)
-
-def launch_web_vanillajs_service(auto_init=True):
-    launch_web_service("vanillajs-web", dirname="web", auto_init=auto_init)  
-
-# Launch all services so that all service resources are registered with Tilt
-
-# Server services
-launch_go_frontend()
-launch_python_frontend()
-launch_ruby_frontend()
-launch_java_frontend()
-launch_dotnet_frontend()
-launch_node_frontend()
-launch_elixir_frontend()
-
-launch_go_message_service()
-launch_python_message_service()
-launch_ruby_message_service()
-launch_java_message_service()
-launch_dotnet_message_service()
-launch_node_message_service()
-launch_elixir_message_service()
-
-launch_go_name_service()
-launch_python_name_service()
-launch_ruby_name_service()
-launch_java_name_service()
-launch_dotnet_name_service()
-launch_node_name_service()
-launch_elixir_name_service()
-
-launch_go_year_service()
-launch_python_year_service()
-launch_ruby_year_service()
-launch_java_year_service()
-launch_dotnet_year_service()
-launch_node_year_service()
-launch_elixir_year_service()
-
-# Client services
-launch_web_vanillajs_service()
-
-# Create map of "groups" of services to commonly run together (e.g. all node services)
-supported_languages = ["go", "py", "rb", "java", "dotnet", "node", "elixir"]
-language_specific_services = ["frontend", "message", "name", "year"]
-
-def append_lang(lang):
-    lang_appended_list = []
-    for service in language_specific_services:
-        lang_appended_list.append(service + "-" +lang)
-    return lang_appended_list
-groups = { i: append_lang(i) for i in supported_languages }
-groups["web"] = ["vanillajs-web"]
-
-# Common resources we always want to run
-resources = ['collector', 'redis', 'curl greeting']
-
-# Create the final list of services to run
-for arg in to_run:
-  if arg in groups:
-    resources += groups[arg]
-  else:
-    # also support specifying individual services instead of groups, e.g. `tilt up a b d`
-    resources.append(arg)
-
-# Enable the specified subset of resources or just all the required services + go services if nothing is specified
-config.set_enabled_resources(resources)
-
-###
-# Notes
-###
-
-# syntax for local_resource:
-# local_resource ( name , build_cmd , deps=None , trigger_mode=TRIGGER_MODE_AUTO , resource_deps=[] , ignore=[] , auto_init=True , serve_cmd='go run cmd/shepherd/main.go -debug' )
-# name
-# command to build the thing to run (empty in our world)
-# deps are a list of files to watch and, if changed, restart the process
-# serve_cmd is the command to run to start the thing, and it's expected that it won't exit
-# eg
-# local_resource ( "shepherd" , "" , serve_cmd='cd cmd/shepherd && go run main.go -debug -p :8081' )
-
-# link to tilt API docs: https://docs.tilt.dev/api.html
+# prints show up in the Tiltfile log so you know what's been run
```

Comment:
Hmmmm I'm worried this will blow away all the history for this file, it seems like there were a few formatting changes made, could we do that separately to this PR?

Reply:
Oh, huh. I have no memory of this place. I've reverted that commit 👍